TVOAppliance/hf_upgrade.sh

```diff
@@ -158,11 +153,13 @@ function install_package()
 	#set the default python3
 	update-alternatives --install /usr/bin/python3 python3 /usr/local/bin/python3.8 0
 
-	#restart the services post install
-	systemctl restart tvault-config wlm-workloads wlm-api wlm-workloads
 
-	#call function - restart wlm-cron service on primary node only.
-	restart_wlm_cron_on_primary_node
+	#restart all active services
+	SERVICE_NAMES=('tvault-config' 'wlm-workloads' 'wlm-api' 'wlm-workloads' 'wlm-cron')
```

Comment:
this list can have the `tvault-object-store` service as well.
the `restart_service` can be call be called after `systemctl daemon-reload`